src/models/event-timeline-set.js

```diff
@@ -243,6 +243,31 @@ EventTimelineSet.prototype.findEventById = function(eventId) {
     });
 };
 
+/**
+ * Find an event by id which is stored in our timelines and replace it with a new event
```

Comment:
nit: capitalize ID

Reply:
This method will be removed again in the next PR, so should be ok.